lib/converter.js

```diff
@@ -1,10 +1,28 @@
 /* global Opal */
 const util = require('util')
-const fs = require('fs')
+const {writeFile, readFile} = require('fs').promises
 const path = require('path')
-const writeFile = util.promisify(fs.writeFile)
 const puppeteer = require('puppeteer')
 const asciidoctor = require('asciidoctor.js')()
+const getPort = require('get-port')
+const handler = require('serve-handler')
+const {createServer} = require('http')
+const debug = util.debuglog('asciidoctor-pdf')
```

Comment:
This way, debug logs appear when doing `NODE_DEBUG=asciidoctor-pdf ./bin/asciidoctorjs-pdf ...`